ft_headmovement.m

```diff
@@ -1,4 +1,4 @@
-function [grad] = ft_headmovement(cfg)
+function [grad, grad_orig, grad_avg, trialinfo] = ft_headmovement(cfg)
```

Comment:
why does this function not accept a data structure (which should contain the HLC channels) as input?